modules/caddyhttp/staticerror.go

```diff
@@ -95,7 +95,7 @@ func (e *StaticError) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 	return nil
 }
 
-func (e StaticError) ServeHTTP(w http.ResponseWriter, r *http.Request, _ Handler) error {
+func (e StaticError) ServeHTTP(w http.ResponseWriter, r *http.Request, h Handler) error {
```

Comment:
this feels like it hurts readability. I agree with what @mholt was saying in an above comment, better keep the `_` for these function whose goal is to fulfill an interface.